extensions/qute/deployment/src/main/resources/dev-templates/preview.html

```diff
@@ -0,0 +1,30 @@
+{#include main}
+  {#style}
+   .annotation {
```

Comment:
Guys, if you're not going to need custom CSS, stop copying this blindly, please? ;)